src/redux/books/books.js

```diff
@@ -1,36 +1,45 @@
+// apiKey= PqQLW0Acp1ia75XdU4Nk
+
 // Action types
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
 
-const initialState = [
-  {
-    id: '1', title: 'The Finished Mystery', author: 'Watchtower Society',
-  },
-  {
-    id: '2', title: 'Repairing Windows', author: 'Scott Muller',
-  },
-  {
-    id: '3', title: 'The African Child', author: 'Camara Laye',
-  },
-];
+const initialState = [];
 
 export const addBook = (book) => ({
   type: ADD_BOOK,
   book,
 });
+
 export const removeBook = (bookId) => ({
   type: REMOVE_BOOK,
   bookId,
 });
 
+export const fetchBooks = () => (dispatch) => fetch(
+  'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/PqQLW0Acp1ia75XdU4Nk/books',
+)
+  .then((response) => response.json())
+  .then((data) => {
+    Object.keys(data).forEach((book) => {
+      dispatch({
+        type: ADD_BOOK,
+        book: {
+          item_id: book,
+          ...data[book][0],
+        },
+      });
+    });
+  });
+
 // reducer
 const booksReducer = (state = initialState, action) => {
   switch (action.type) {
     case ADD_BOOK:
       return [...state, action.book];
 
     case REMOVE_BOOK:
-      return state.filter((book) => book.id !== action.bookId);
+      return state.filter((book) => book.item_id !== action.bookId);
```

Comment:
- [ ] Kindly be noticed that you should create a new action type to get fetch the books ie: 
```
const FETCH_BOOK = 'bookStore/books/FETCH_BOOK';
```
![Screenshot from 2022-06-16 00-21-16](https://user-images.githubusercontent.com/33752735/173942198-a271c74a-acc1-4463-9a53-f7785ca4167b.png)